tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/interpreter/TornadoVMInterpreter.java

```diff
@@ -950,6 +974,29 @@ public void clearInstalledCode() {
         Arrays.fill(installedCodes, null);
     }
 
+    /**
+     * Container class that holds information about object allocation counts.
+     * Used to track the number of persistent objects and the number of objects
+     * that need to be allocated.
+     */
+    private static class ObjectAllocationInfo {
```

Comment:
I think you can use a Java `record` since they are both final

Reply:
This is fixed now